### ISSUE TITLE
Resource Portal download survey

### ISSUE THREAD

**Issue by IAlferink** [digital-york/ncelp#166]
**Change two things at download survey:**
1. Make country compulsory
2. Add two options to tick box:
- NCELP Specialist Teacher
- NCELP Hub school teacher

**Change expire session time**
- Change cookie scope from session to 30 days.

**Comment by paluchas** [comment]
Test  slack - @paluchas look at this

**Comment by paluchas** [comment]
@paluchas test commenting on slack

**Comment by IAlferink** [comment]
@yf508 Could you please implement these changes?

**Comment by IAlferink** [comment]
@yf508 Any updates on this?

**Comment by paluchas** [comment]
@IAlferink The updated version should be available on the test server. Follow is a screenshot of how this looks. I have some questions:
- Is the capitalisation of the fields name right?
- Is the localisation of new checkbox fields in the options tree right? Or should I move this up after "Media / journalist" option?
- Is the wording changed in Country label as you would expect? E.g. I have added "(this is obligatory in order to download a resource)"

![Screenshot 2019-08-20 at 15 56 29](https://user-images.githubusercontent.com/1863245/63358927-bfe2c800-c363-11e9-9987-4239e0b5f8c0.png)

**Comment by IAlferink** [comment]
@paluchas 

- Is the capitalisation of the fields name right?
_No, please change to: **NCELP Hub School Teacher**. This would then be in line with language elsewhere on the website._

- Is the localisation of new checkbox fields in the options tree right? Or should I move this up after "Media / journalist" option?
_Please, move after "Media / journalist" option_

- Is the wording changed in Country label as you would expect? E.g. I have added "(this is obligatory in order to download a resource)"
_Yes, that is fine_

**Comment by websnacks** [comment]
@IAlferink 

Hi, I made the changes. Please have a look at [dev-resources.ncelp.org](https://dev-resources.ncelp.org)

**Comment by IAlferink** [comment]
@websnacks Thanks! Looks good. Could you make one more change, please? (I think you actually suggested this a while ago :-)

Could we change this:
**About you - Please tell us what your current role(s) is / are. Tick at least one field (this is obligatory in order to download a resource):**

To:
**About you**
Please tell us what your current role(s) is / are. Tick at least one field (this is obligatory in order to download a resource):

**Comment by websnacks** [comment]
Yes I made that change now too. I also made sure that all elements including the Save & Submit buttons are lined up.

**Comment by IAlferink** [comment]
It looks much cleaner. Thank you!

**Comment by IAlferink** [comment]
@paluchas @yf508 
Can you please look at this (from description):
**Change expire session time**

- Change cookie scope from session to 30 days.

I just tried to download 2 things in a row (same session, same resource) and got asked to fill in the survey twice.

**Comment by yf508** [comment]
The problem should not be caused by the expire session time. I've checked - the expire session time has been configured to 30 days. 

If there are more than 1 files downloadable for a single resource, click 'Download' for the first one will popup  the survey form, after saved the survey form and download the file, currently there is no way for the parent page (resource details page) to know that the survey form has been filled and saved. I've made some changes to refresh the resource details page after 'Download' or 'Close' button being clicked. pls check if it works as expected on test. @IAlferink
